javascript/js-callback-functions_user-login-callback-4-with-multiple-functions/js/index.js

```diff
@@ -9,3 +9,14 @@ function handleUserLogin(onSuccess, onError, userName) {
 }
 
 // The exercise starts here!
+
+function showWelcomeMessage (userName,userRole) {
+  console.log(`Welcome ${userName}! U logged in as ${userRole}.`)
+};
+
+function showErrorMessage (errorMessage) {
+  console.log(`Login error: ${errorMessage}`)
+};
+
+handleUserLogin(showWelcomeMessage, showErrorMessage, 'Jane Doe');
+handleUserLogin(showWelcomeMessage, showErrorMessage, 'John Doe');
```

Comment:
Why calling it twice ?
Otherwise you are good to Go!

Reply:
to console log the different messages according to the username given